sentry_logging/lib/src/logging_integration.dart

```diff
@@ -0,0 +1,58 @@
+import 'dart:async';
+
+import 'package:logging/logging.dart';
+import 'package:sentry/sentry.dart';
+import 'extension.dart';
+
+/// An [Integration] which listens to all messages of the
+/// [logging](https://pub.dev/packages/logging) package.
+class LoggingIntegration extends Integration<SentryOptions> {
+  /// Creates the [LoggingIntegration].
+  ///
+  /// Setting [logExceptionAsEvent] to true (default) captures all
```

Comment:
This should be `minimalEventLevel`, similar to the [C# Serilog Integration](https://docs.sentry.io/platforms/dotnet/guides/serilog/).

Reply:
Also a `minimumLogLevel` would be nice.

Reply:
we should have the min. for event and breadcrumb, see Timber https://github.com/getsentry/sentry-java/blob/main/sentry-android-timber/src/main/java/io/sentry/android/timber/SentryTimberIntegration.kt#L18-L19